src/components/EditListingServiceTypePanel/EditListingServiceTypePanel.js

```diff
@@ -73,7 +74,7 @@ const EditListingServiceTypePanel = (props) => {
           if (newTitle !== title) {
             updateValues.publicData.prices = null
           }
-
+          updateValues.publicData.keywords = serviceTypeKeywords[newTitle]
```

Comment:
I know I was Mr. Optimistic yesterday (long day), but let's at least fallback to an empty array here so that code down the line won't blow up if our `newTitle` key is for some reason not found in `serviceTypeKeywords`. I'd say let's do a `console.error` and then fallback to empty array for safety and let the code continue on.

Reply:
Definitely! Good call

Where exactly did you want to do a console.error though? 

My thoughts were that the change would just look like this 

`updateValues.publicData.keywords = serviceTypeKeywords[newTitle] || []`

Reply:
Something like this:
```suggestion
          const keywords = serviceTypeKeywords[newTitle];
          
          if (!keywords) {
            console.error(‘No keywords found for this serviceType’);
          }
          
          updateValues.publicData.keywords = keywords || [];
```